README.md

```diff
@@ -25,7 +25,11 @@ K6_PROMETHEUS_REMOTE_URL=http://localhost:9090/api/v1/write ./k6 run script.js -
 
 Add TLS and HTTP basic authentication:
 ```
-K6_PROMETHEUS_REMOTE_URL=https://localhost:9090/api/v1/write K6_PROMETHEUS_INSECURE_SKIP_TLS_VERIFY=false K6_CA_CERT_FILE=example/tls.crt K6_PROMETHEUS_USER=foo K6_PROMETHEUS_PASSWORD=bar ./k6 run script.js -o output-prometheus-remote
+K6_PROMETHEUS_REMOTE_URL=https://localhost:9090/api/v1/write \
+K6_PROMETHEUS_INSECURE_SKIP_TLS_VERIFY=false \
+K6_PROMETHEUS_CA_CERT_FILE=example/tls.crt \
```

Comment:
Do we need this? The user could use the Go env vars `SSL_CERT_FILE` and `SSL_CERT_DIR` for passing the location or the dir of the root store.

Reply:
It's a good question :thinking: 

Honestly, I think that we can stick with the default golang's environment variable if that works precisely the same way, but I guess it's still important to mention that because maybe not all of our users are golang devs. But I'm not so strict in that opinion.

Reply:
@yorugac WDYT?

Reply:
Apologies for the delay! This message totally got lost in my Github subscriptions.
This env var appeared in order to connect to Prometheus set up with [auth](https://prometheus.io/docs/prometheus/latest/configuration/https/): if there is another way to connect to such an instance without using env var, it should be checked. 

Btw, Mimir has something similar in its setup to Prometheus: https://grafana.com/docs/mimir/latest/operators-guide/secure/securing-communications-with-tls/#configure-tls-certificates-in-grafana-mimir

Additionally, IIRC, I had a reasoning that one might want to keep specific certificates separately, and external env vars like `SSL_CERT_FILE` are not explicit enough and / or may not be available. This reasoning is debatable and a matter of product design, of course.

Hope that helps!

Reply:
Thanks, @yorugac. If I'm understanding well we don't have a specific requirement for it; then it sounds reasonable to me to keep the k6 and the output config consistent and with as less as possible options for reducing the risk of breaking change in the future. I would be happy to bring back this if we will receive an explicit request for improving the UX on this front.

>if there is another way to connect to such an instance without using env var, it should be checked.

 There is and it's even better than env var because it is just implicit and it is the system trusted certs store. We are doing the same with the k6 core. https://github.com/grafana/k6/issues/218#issuecomment-301127630

Reply:
Yes, sounds good to me :+1: